### PR TITLE
Created bottom-bar component

### PR DESCRIPTION
**Pull request by tiagoboeing**
- [x] Created `bottom-bar` component

### CODE REVIEW

File: src/components/bottom-bar/bottom-bar.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+})
+export class BottomBar {
+  @Prop({ reflect: true }) theme: TecnologiaTheme = defaultTheme;
+
+  @Prop() text: string;
+
+  @Prop() buttonPosition = ButtonPosition.left;
+
+  @Prop({ mutable: false, reflect: false })
+  useAnimation = true;
```
Comment (lucasreichert3) [inline src/components/bottom-bar/bottom-bar.tsx:19]: I tried use false to this prop, but didn't work.

https://i.imgur.com/HYznK5R.gif

Comment (tiagoboeing) [reply src/components/bottom-bar/bottom-bar.tsx:19]: On vanilla JS to `false` values you shouldn't pass. For example, on Angular using one way data binding that will be work.

Comment (tiagoboeing) [reply src/components/bottom-bar/bottom-bar.tsx:19]: Maybe the reflect property should resolve, i will test:

image

https://stenciljs.com/docs/properties

Comment (tiagoboeing) [reply src/components/bottom-bar/bottom-bar.tsx:19]: I tested on CodeSandbox and works. The animation class was not added to element: https://codesandbox.io/s/tecnologia-webcomponents-demo-p6tht?file=/index.html

image

**Pull request merged**